setup.sh

```diff
@@ -34,111 +64,115 @@ sudo apt-get install -y libusb-1.0-0-dev libreadline-dev libncurses-dev
 
 # Build I2S kernel module
 PI_MODEL=$(cat /proc/device-tree/model | awk '{print $3}')
-RPI4B_FLAG=""
-if [ $PI_MODEL == "4" ] ; then
-    RPI4B_FLAG="-DRPI_4B"
+if [[ $PI_MODEL = 4 ]]; then
+  I2S_MODULE_CFLAGS="-DRPI_4B"
+  SEP=" "
 fi
 
-if [ $# -ge 1 ] && [ $1 = "xvf3510" ] ; then
-    pushd $RPI_SETUP_DIR/loader/i2s_master > /dev/null
-    I2S_MASTER_FLAG="-DI2S_MASTER"
+case $I2S_MODE in
+  master)
+    I2S_MODULE_CFLAGS="$I2S_MODULE_CFLAGS$EP-DI2S_MASTER"
```

Comment:
A space is needed in I2S_MODULE_CFLAGS when two flags are used

Reply:
Ah yes. Fixed.